easybib/recipes/role-couchdb.rb

```diff
@@ -1 +1,3 @@
+include_recipe 'ies::role-generic'
 include_recipe 'apache-couchdb'   # default.rb includes configure.rb, which includes service.rb
+include_recipe 'monit::service'
```

Comment:
Yes, it is. But I am wondering if it would make even more sense to place it in https://github.com/till/easybib-cookbooks/blob/master/ies/recipes/role-generic.rb to enable it for all instances. Dont see why not, but could cause more noise.